css/style.css

```diff
@@ -1347,4 +1256,133 @@ html.dark-mode .hidden-light {
 }
 html.dark-mode .hidden-dark {
   display: none;
+}
+
+@media all and (max-width: 1064px) {
+  header {
+    column-gap: 12px;
+    padding-inline: 32px;
+  }
+
+  #mobile-menu {
+    display: block;
+  }
+
+  .algolia-autocomplete {
+    display: none !important;

```

Comment:
Remove important

Reply:
This `!important` is necessary since the styles for the search are injected by an external source.